app/views/comments/_form.html.erb

```diff
@@ -1,8 +1,11 @@
 <%= form_with model: [ @article, @article.comments.build ] do |form| %>
   <p>
     <%= form.label :commenter, class:'d-flex justify-content-center' %>
-    <%= form.text_field :commenter, class: 'form-control mx-auto', style:'width: 15rem;'%>
-  </p>
+    <% if user_signed_in? %>
+      <%= form.text_field :commenter,value: current_user.email, readonly:true,class: 'form-control mx-auto', style:'width: 15rem;'%> 
```

Comment:
`<%= form.text_field :commenter, value: current_user.email, readonly: true, class: 'form-control mx-auto', style:'width: 15rem;'%>`

Reply:
We must don't put CSS code in the HTML